MapboxVision/Services/Recording/VideoTrimmer.swift

```diff
@@ -41,23 +36,28 @@ final class VideoTrimmer {
             return
```

Comment:
`completion` is not called for else-branch of guard statement here and line 36 above.
IMHO we should call it.

Reply:
Good catch. Will update.